tests/i18n/tests.py

```diff
@@ -1472,12 +1472,32 @@ def test_parse_language_cookie(self):
         r.META = {'HTTP_ACCEPT_LANGUAGE': 'de'}
         self.assertEqual(g(r), 'zh-hans')
 
+    @override_settings(
+        LANGUAGES=[
+            ('en', 'English'),
+            ('de', 'German'),
+            ('de-at', 'Austrian German'),
+            ('pl', 'Polish'),
+        ]
+    )
     def test_get_language_from_path_real(self):
         g = trans_real.get_language_from_path
         self.assertEqual(g('/pl/'), 'pl')
         self.assertEqual(g('/pl'), 'pl')
         self.assertIsNone(g('/xyz/'))
 
+        self.assertEqual(g('/en/'), 'en')
+        self.assertEqual(g('/en-gb/'), 'en')
+
+        self.assertEqual(g('/de/'), 'de')
+        self.assertEqual(g('/de-at/'), 'de-at')
+        self.assertEqual(g('/de-ch/'), 'de')
+
+        # Regresion test for #27063
```

Comment:
Describe the issue in a comment rather than including a ticket reference (saves a lookup step to understand the purpose of the test).